### ISSUE TITLE
Directories with special chars returns Not found

### ISSUE THREAD

**Issue by djalmaaraujo** [zeit/micro-list#7]
To continue the conversation from: https://github.com/zeit/micro-list/issues/5.

**Comment by rauchg** [comment]
We could also use punycode, looks a bit nicer than `encodeURIComponent`:

```
> require('punycode').encode('presentación')
'presentacin-zeb'
```

**Comment by djalmaaraujo** [comment]
@rauchg OK, tested now with ```decodeURIComponent``` and worked. Submitted the PR soon.

**Comment by djalmaaraujo** [comment]
Fixed on #8
